real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/messages/LogicExceptionMeasurment.java

```diff
@@ -0,0 +1,42 @@
+package com.roboclub.robobuggy.messages;
+
+import com.roboclub.robobuggy.main.MESSAGE_LEVEL;
+import com.roboclub.robobuggy.ros.Message;
+
+public class LogicExceptionMeasurment extends BaseMessage implements Message{
+	String message;
+	MESSAGE_LEVEL level;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1263621995/issues/source?bid=2938665&fileBranchId=2939970#l8)